locales/es.json

```diff
@@ -46,170 +46,170 @@
-   "Compiling...": "Compiling...",
-   "Building...": "Building...",
-   "gas at the price of": "gas at the price of",
-   "Wei, estimated cost:": "Wei, estimated cost:",
-   "using": "using",
-   "Embark is building, please wait...": "Embark is building, please wait...",
-   "instantiated js-ipfs object configured to the current environment (available if ipfs is enabled)": "instantiated js-ipfs object configured to the current environment (available if ipfs is enabled)",
-   "Initializing Embark Template...": "Initializing Embark Template...",
-   "Name your app (default is %s):": "Name your app (default is %s):",
-   "Unrecognized balance string \"%s\"": "Unrecognized balance string \"%s\"",
-   "{{className}} has a dependency on {{dependencyName}}": "{{className}} has a dependency on {{dependencyName}}",
-   ", but it is not set to deploy. It could be an interface.": ", but it is not set to deploy. It could be an interface.",
-   "Done. %s generated": "Done. %s generated"
+   "language to use (default: en)": "lenguaje a utilizar (Por defecto: es)",
+   "download template": "descargar plantilla",
+   "Start the Embark console": "Iniciar la consola de Embark",
+   "When set, will print the gas cost for each contract deploy": "Al ser establecido, imprimirá el costo en gas para cada contrato desplegado",
+   "Graph will not include undeployed contracts": "Graph no incluirá los contratos no desplegados",
+   "Graph will not include functions": "Graph no inclurá las funciones",
+   "Graph will not include events": "Graph no inclurá los eventos",
+   "ENS domain to associate to": "dominio ENS a ser asociado",
+   "output usage information and help information": "imprime información de uso e información de ayuda",
+   "Starting %s process": "Iniciando proceso %s",
+   "Starting Blockchain node in another process": "Iniciando nodo Blockchain en otro proceso",
+   "Blockchain node is ready": "Nodo Blockchain esta listo",
+   "Storage process for {{storageName}} ended before the end of this process. Code: {{code}}": "Proceso de almacenamiento para {{storageName}} terminó antes del final de este proceso. Código: {{code}}",
+   "ipfs process started": "proceso ipfs iniciado",
+   "Compiling...": "Compilando...",
+   "Building...": "Construyendo...",
+   "gas at the price of": "gas al precio de",
+   "Wei, estimated cost:": "Wei, costo estimado:",
+   "using": "utilizando",
+   "Embark is building, please wait...": "Embark esta construyendo, por favor espere...",
+   "instantiated js-ipfs object configured to the current environment (available if ipfs is enabled)": "objeto instanciado js-ipfs se configuró para el entorno actual (disponible si ipfs esta habilitado)",
+   "Initializing Embark Template...": "Inicializando plantilla de Embark...",
+   "Name your app (default is %s):": "Da un nombre a tu aplicación (Por defecto es %s):",
+   "Unrecognized balance string \"%s\"": "Saldo no reconocido en string \"%s\"",
```

Comment:
"Cadena de saldo no reconocida \"%s\""

Reply:
Hi,

I think you're wrong interpreting the texts. Are you aware of the context where the text is being used within the source code ?